jobs/quoscient_cybersecurity_germany_permanent.html

```diff
@@ -0,0 +1,57 @@
+---
+title: Senior Python Developer
+company: QuoScient GmbH
+url: http://quoscient.io
+location: Frankfurt am Main, Germany
+contract: permanent
+contact:
+    name: Ina Klemm
+    email: careers@quoscient.io
+created: !!timestamp '2017-03-01' 
+tags:
+  - germany
+  - python 3
+  - cyber security
+  - NoSQL
+  - senior
+  - OOP
+  - Git
+  - Unix
+---
+
+# Overview
+
+QuoScient is a specialist in Information Security Services. We are providing security integration support, incident response consultancy and a distributed analysis platform that is supporting collaboration and information sharing.  
+ 
+We support customers with understanding, responding and also anticipating threats, targeting them, their clients and industry peers. We develop collaborative tools and methods enabling teams to monitor, detect and counter these threats more efficiently and anticipatively.   
+ 
+To support that vision QuoScient is implementing a state of the art distributed platform where threat and incident information meet a synergy of security operation tools from malicious artifact analysis to threat intelligence modeling
+
+# Role
+
+To strengthen our Development Team based in Frankfurt, we are looking for experienced  
+ 
+Senior Python developers (m/f)  
```

Comment:
(m/f)  - this looks a bit odd. It might be better to say at the end that you are an equal opportunity employer, which generally means that you are open to all genders, age, ethnicity etc.